http-protocol-binding.md

```diff
@@ -213,16 +213,72 @@ and per [RFC7230, section 3][rfc7230-section-3], HTTP headers MUST only use
 printable characters from the US-ASCII character set, and are terminated by a
 CRLF sequence with OPTIONAL whitespace around the header value.
 
-String values MUST be percent-encoded as described in [RFC3986, section
-2.4][rfc3986-section-2-4] before applying the header encoding rules described in
-[RFC7230, section 3.2.6][rfc7230-section-3-2-6].
-
-When decoding an HTTP message into a CloudEvent, these rules MUST be applied in
-reverse -- [RFC7230, section 3.2.6][rfc7230-section-3-2-6] decoding to an ASCII
-string, and then a **single round** of percent-decoding as described in
-[RFC3986, section 2.4][rfc3986-section-2-4] to produce a valid UTF-8 String.
-(Note that applying percent-decoding an incorrect number of times can result in
-message corruption or security issues.)
+When encoding a CloudEvent as an HTTP message, string values MUST be
+percent-encoded as described below. This is compatible with
+[RFC3986, section 2.1][rfc3986-section-2-1] but is more specific
+about what needs encoding. The resulting string does not then need
+any further quoting, due to space and quote characters being encoded.
+
+When decoding an HTTP message into a CloudEvent, quoted strings as
+described in [RFC7230, section 3.2.6][rfc7230-section-3-2-6] MUST be
+decoded to an ASCII string, and then a **single round** of
+percent-decoding as described below. HTTP headers for CloudEvent
+attribute values do not support parenthetical comments, so the
+initial decoding only needs to handle double-quoted values, including
+backslash escapes. Header values produced via the percent-encoding
+described here will never include double-quoted values, but they MUST
+be supported for compatibility with older versions of this
+specification which did not require double-quote and space
+characters to be percent-encoded.
```

Comment:
That's the sentece that (I think) will allow old and new sdks to communicate. I like it.

Just, can you specify that _they MUST be supported when receiving events_ or something like that? Just to make clear the first part of the sentence is about producing, the second is about consuming

Reply:
Yup, happy to do so. (Will do that in a separate commit, hopefully some time in the next few hours.)

Reply:
Perfect!